relay-metrics/src/aggregator.rs

```diff
@@ -71,13 +71,9 @@ impl BucketKey {
     ///
     /// This is used for partition key computation and statsd logging.
     fn hash64(&self) -> u64 {
-        BucketKeyRef {
-            project_key: self.project_key,
-            timestamp: self.timestamp,
-            metric_name: &self.metric_name,
-            tags: &self.tags,
-        }
-        .hash64()
+        let mut hasher = FnvHasher::default();
```

Comment:
`BucketKeyRef` is no longer required. The hash created here and the hash used for partitioning do not have to be the same, which is why we can hash differently.